{"success": false, "error": "Traceback (most recent call last):\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 544, in _run_solc\n    ret: Dict = json.loads(stdout)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/__init__.py\", line 357, in loads\n    return _default_decoder.decode(s)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/decoder.py\", line 337, in decode\n    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/decoder.py\", line 355, in raw_decode\n    raise JSONDecodeError(\"Expecting value\", s, err.value) from None\njson.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/slither/__main__.py\", line 826, in main_impl\n    ) = process_all(filename, args, detector_classes, printer_classes)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/slither/__main__.py\", line 86, in process_all\n    compilations = compile_all(target, **vars(args))\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 637, in compile_all\n    compilations.append(CryticCompile(target, **kwargs))\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 117, in __init__\n    self._compile(**kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 548, in _compile\n    self._platform.compile(self, **kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 154, in compile\n    targets_json = _get_targets_json(compilation_unit, self._target, **kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 276, in _get_targets_json\n    return _run_solc(\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 548, in _run_solc\n    raise InvalidCompilation(f\"Invalid solc compilation {stderr}\")\ncrytic_compile.platform.exceptions.InvalidCompilation: Invalid solc compilation /Users/giacomoibba/Desktop/SmarthER/mainnet/d5/d525beaBF69d72e170d6f9ee153D3F6F70F7D177_DefiPassportClaimer.sol:4:1: Error: Source \"/Users/giacomoibba/Desktop/SmarthER/lib/Address.sol\" not found: File not found.\nimport {Address} from \"../../lib/Address.sol\";\n^--------------------------------------------^\n/Users/giacomoibba/Desktop/SmarthER/mainnet/d5/d525beaBF69d72e170d6f9ee153D3F6F70F7D177_DefiPassportClaimer.sol:5:1: Error: Source \"/Users/giacomoibba/Desktop/SmarthER/lib/Ownable.sol\" not found: File not found.\nimport {Ownable} from \"../../lib/Ownable.sol\";\n^--------------------------------------------^\n/Users/giacomoibba/Desktop/SmarthER/mainnet/d5/d525beaBF69d72e170d6f9ee153D3F6F70F7D177_DefiPassportClaimer.sol:6:1: Error: Source \"/Users/giacomoibba/Desktop/SmarthER/debt/sapphire/ISapphireCreditScore.sol\" not found: File not found.\nimport {ISapphireCreditScore} from \"../../debt/sapphire/ISapphireCreditScore.sol\";\n^--------------------------------------------------------------------------------^\n/Users/giacomoibba/Desktop/SmarthER/mainnet/d5/d525beaBF69d72e170d6f9ee153D3F6F70F7D177_DefiPassportClaimer.sol:7:1: Error: Source \"/Users/giacomoibba/Desktop/SmarthER/debt/sapphire/SapphireTypes.sol\" not found: File not found.\nimport {SapphireTypes} from \"../../debt/sapphire/SapphireTypes.sol\";\n^------------------------------------------------------------------^\n/Users/giacomoibba/Desktop/SmarthER/mainnet/d5/d525beaBF69d72e170d6f9ee153D3F6F70F7D177_DefiPassportClaimer.sol:8:1: Error: Source \"/Users/giacomoibba/Desktop/SmarthER/mainnet/d5/IDefiPassport.sol\" not found: File not found.\nimport {IDefiPassport} from \"./IDefiPassport.sol\";\n^------------------------------------------------^\n/Users/giacomoibba/Desktop/SmarthER/mainnet/d5/d525beaBF69d72e170d6f9ee153D3F6F70F7D177_DefiPassportClaimer.sol:230:1: Error: Source \"/Users/giacomoibba/Desktop/SmarthER/mainnet/d5/SapphireTypes.sol\" not found: File not found.\nimport {SapphireTypes} from \"./SapphireTypes.sol\";\n^------------------------------------------------^\n\n", "results": {}}
